packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -535,7 +589,7 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
             options={
                 {
                     id: this.props.traceId + this.props.outputDescriptor.id + 'focusContainer',
-                    height: parseInt(this.props.style.height.toString()) - this.getMarkersLayerHeight(),
+                    height: parseInt(this.props.style.height.toString()) - this.getMarkersLayerHeight() - (document.getElementById(this.props.traceId + this.props.outputDescriptor.id + 'searchBar')?.offsetHeight ?? 0),
```

Comment:
eslint warning:
This line has a length of 218. Maximum allowed is 180.eslint[max-len](https://eslint.org/docs/rules/max-len)